Makefile

```diff
@@ -69,3 +70,14 @@ e2e-bootstrap:
 .PHONY: e2e-azure
 e2e-azure:
 	bats -t test/bats/azure.bats
+
+local-e2e-bootstrap:
+# TODO RUN LOCAL E2E
```

Comment:
Is there still something to do here? If not, replace this `# TODO` with a relevant comment.